server/src/main/java/org/elasticsearch/indices/recovery/RecoveryTranslogOperationsRequest.java

```diff
@@ -87,8 +88,11 @@ public RetentionLeases retentionLeases() {
         return retentionLeases;
     }
 
-    @Override
-    public void readFrom(StreamInput in) throws IOException {
+    public long mappingVersion() {
```

Comment:
can you document what mapping version that tis?

Reply:
I pushed https://github.com/elastic/elasticsearch/pull/41099/commits/00eb6d9f5173944f8b0336c830f3433ac2fb9f92